x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -38523,13 +38523,15 @@
     "xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.apiKeyOwnerTitle": "Propriétaire de la clé d'API",
     "xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.deleteAriaLabel": "Supprimer",
     "xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.deleteButtonTooltip": "Supprimer",
-    "xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.durationTitle": "Temps passé sur l'exécution de la règle (mm:ss).",
+    "xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.durationTitle": "Durée",
+    "xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.durationTitleTooltip": "Temps passé sur l'exécution de la règle (mm:ss).",
```

Comment:
You do not need to add these translation because at each release, we have a process to rebuild all these translations. So, It is ok to remove it, you only get error when you have duplicate key.

Reply:
We just had an exact translations we need for that, so I decided to save translators time. And also for testing (to see that we actually have everuthing translated).
Do you think it's safer to delete them for now?

Reply:
Didn't knew you speak French, Japanese and Chinese! :P

I would say yes, you should remove them but I'm not 100% sure cc @XavierM

Reply:
@jcger I have a lot of hidden talents :-P.

I've fixed it. Probably you are right. It's safer. And I've alredy tested it with translations.